tracer/src/Datadog.Trace.Tools.Runner/Program.cs

```diff
@@ -58,6 +58,18 @@ internal static int Main(string[] args)
             AppDomain.CurrentDomain.ProcessExit += (_, _) => CurrentDomain_ProcessExit(applicationContext);
             AppDomain.CurrentDomain.DomainUnload += (_, _) => CurrentDomain_ProcessExit(applicationContext);
 
+            if (applicationContext.Platform == Platform.Linux)
+            {
+                // Make dd-dotnet executable
+                var ddDotnet = Utils.GetDdDotnetPath(applicationContext);
+
+                if (ddDotnet != null && File.Exists(ddDotnet))
+                {
+                    // Make sure the dd-dotnet binary is executable
+                    System.Diagnostics.Process.Start("chmod", $"+x {ddDotnet}")!.WaitForExit();
```

Comment:
Doesn't this need `sudo` to work properly in non-ci environments?

Reply:
I don't think you need sudo permissions to make a file executable

Reply:
yeah, you're right, as long as you own the executable, which is likely here. TIL